sonic-thermalctld/scripts/thermalctld

```diff
@@ -781,6 +802,7 @@ class ThermalControlDaemon(daemon_base.DaemonBase):
 
         self.thermal_monitor = ThermalMonitor(self.chassis)
         self.thermal_monitor.task_run()
+        self.thermal_monitor.init_tables()
```

Comment:
why we need this line?

Reply:
this is not a formal fix. 
it is to simulate the original flow and guarantee the only difference is the socket not shared between parent and child and demonstrate the principle of a potential fix.